models/experimental/functional_unet/tt/unet_shallow_ttnn.py

```diff
@@ -195,6 +194,7 @@ def __init__(
         conv_cache={},
         should_reshard=False,
```

Comment:
What happens if we make this true for all upblock/downblocks? Does it just do the right thing? 

If yes, it might make sense to default to `should_reshard=True` and no longer specify it when we instantiate the up/downblocks.

Reply:
Seems fine :)